pkg/osv/osv.go

```diff
@@ -28,7 +28,7 @@ const (
 	// maxQueriesPerRequest splits up querybatch into multiple requests if
 	// number of queries exceed this number
 	maxQueriesPerRequest  = 1000
-	maxConcurrentRequests = 25
+	maxConcurrentRequests = 1000
```

Comment:
I feel 1000 might be too much. how long to run with maxConcurrentRequests set to 100?

Reply:
Discussed offline, 1000 should be ok to use and doesn't have any disadvantages.